packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -368,6 +367,7 @@ export default class Meetings extends WebexPlugin {
     onReady() {
       this.webex.once(READY, () => {
         StaticConfig.set(this.config);
+        LoggerProxy.set(this.webex.logger);
```

Comment:
just out of curiosity: why do we need this change? and will that cause any issues with logging before onReady is called?

Reply:
its from another PR , which i had rebased to fix test . should not be there now

Reply:
Have removed it for now , Previously wanted to fix it  here

Team will work on it separately , Wont be able to merge this PR unless the other one is done

Reply:
where is this other PR? Is there a jira?